docker-compose.yml

```
@@ -0,0 +1,7 @@
+version: "3.8"
+services:
+  web:
+    container_name: memory_game
+    image: devops_project/memory_game:latest
```

```suggestion
    image: inamit323colman/memory_game:latest
```

This will be our repository name in docker hub